experiments/ClimaEarth/components/land/climaland_bucket.jl

```diff
@@ -214,9 +217,9 @@ function BucketSimulation(
 
     # Add diagnostics
     if use_land_diagnostics
```

Comment:
`output_writer` needs to be defined in the else branch too

Reply:
Thanks for catching that! It's fixed now.